src/styles/_catalog.scss

```diff
@@ -14,6 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition-property: color, transform;
+    transition-duration: 300ms;
```

Comment:
specify for which props transition durations applied
use var for time, it used more than once

Reply:
I don't quite understand what I need to do(((

Reply:
it's fine but u can use more short version like:
transition: color 300ms, transform 300ms